lib/src/base-styles/_nx-page-layout.scss

```diff
@@ -35,11 +35,11 @@
   // In traditional layouts that use plain nx-page-content, that element manages everything below the second
   // row of this template.
   grid-template:
-    'old-header     old-header  old-header' auto
-    'global-sidebar   notices      notices' auto
-    'global-sidebar   header        header' auto
-    'global-sidebar old-sidebar    content' 1fr /
-     auto    $old-sidebar-col-width    1fr;
+    'old-header        old-header          old-header  old-header' auto
+    'global-sidebar     notices             notices       notices' auto
+    'global-sidebar      header              header        header' auto
+    'global-sidebar global-sidebar-2       old-sidebar    content' 1fr /
+     auto                 auto       $old-sidebar-col-width   1fr;
```

Comment:
Any chance you could explain these changes? (I'm somewhat struggling to follow how this grid-template influences the layout of the new sidebar)

Reply:
> Any chance you could explain these changes? (I'm somewhat struggling to follow how this grid-template influences the layout of the new sidebar)

So our overall template has 4 rows and 4 columns with these changes, though not all of them would be used at the same time.  The new sidebar inhabits the grid area named "global-sidebar-2".  In proper usage, you wouldn't also have the old global sidebar, the column to the left of this would be empty and thus sized to zero.  So it would have the header and notices above it, and the inner sidebar and main content area to its right.  Note that the old global sidebar, which uses the "global-sidebar" grid area, goes up alongside the header and notices, not underneath them like the new global sidebar.  Also note that in this template, "old-header" means the _old old_ header for the legacy page layouts (as seen in the layout of the RSC gallery itself), not the soon-to-be-old global sidebar header.

Reply:
To be even more clear, what these changes add is the second column in the template, which has a width of `auto` meaning it will size to its content or zero if it has no distinct content (when the new sidebar is not being used).

Reply:
Thanks for the explanation that's much clearer (the sizing to 0 when not in-use was the main thing I was missing :+1:  )